dev/ci/ci-common.sh

```diff
@@ -115,7 +115,7 @@ git_download()
     git clone "$giturl" "$dest"
     pushd "$dest"
     git checkout "$ref"
-    git log -n 1
+    git --no-pager log -n 1
```

Comment:
should also do the git logs below too
or make the `export GIT_PAGER=` in ci-wrapper unconditionnal.

Reply:
I think probably the thing to do is just make the export unconditional